static/js/message_edit.js

```diff
@@ -322,6 +322,14 @@ function start_edit_maintaining_scroll(row, content) {
 
 exports.start = function (row, edit_box_open_callback) {
     var message = current_msg_list.get(rows.id(row));
+    if (message.raw_content) {
+        start_edit_maintaining_scroll(row, message.raw_content);
+        if (edit_box_open_callback) {
+            edit_box_open_callback();
+        }
+        return;
```

Comment:
There are 3-4 lines here duplicated with below that we should probably move into a function.

Otherwise looks good!